.gitignore

```
@@ -0,0 +1,8 @@
+.terraform
+terraform.tfstate
+*.tfstate*
+terraform.tfvars
+
+# IDEA setting files
+.idea/
+
```

```suggestion
terraform-aws-vpc.iml
```

IDEAの設定にもよるけど、こんなファイルが生成される事があるからこれもGitの管理対象外にしてもらえると:cat: